pkg/api/v1/types.go

```diff
@@ -1328,6 +1337,8 @@ type NodeSystemInfo struct {
 	ContainerRuntimeVersion string `json:"containerRuntimeVersion" description:"Container runtime version reported by the node through runtime remote API (e.g. docker://1.5.0)"`
 	// Kubelet version reported by the node
 	KubeletVersion string `json:"kubeletVersion" description:"Kubelet version reported by the node"`
+	// Open ports on Kubelet
+	KubeletPorts KubeletPorts `json:"kubeletPorts" description:"Information about ports open by Kubelet"`
```

Comment:
I didn't mean that ports should go into Addresses. The different structures of the two fields (struct of fields vs. array of structs) is very different, and it's unclear which address(es) the ports are exposed on. We don't need to address that now, however.

You did what I wanted, thanks.